From: Wolfgang Bauer <wbauer@tmo.at>
Date: Sat, 26 Mar 2016 14:46:19 +0000
Subject: [showdesktop] Fix sizing of the plasmoid in the panel
X-Git-Url: http://quickgit.kde.org/?p=kdeplasma-addons.git&a=commitdiff&h=e164ae7c02f5579e02d26842bee6a4a450e01871
---
[showdesktop] Fix sizing of the plasmoid in the panel

Since 5.6, if you add a "Show Desktop" plasmoid to the panel, the
panel's layout gets completely broken (apparently the plasmoid is sized
to infinity).
This sets the minimum size to sane values which fixes the problem.

(Note that this is for the 5.6 branch only, the problem doesn't exist in
master)

FIXED-IN: 5.6.1
REVIEW: 127499
---


--- a/applets/showdesktop/package/contents/ui/main.qml
+++ b/applets/showdesktop/package/contents/ui/main.qml
@@ -33,8 +33,8 @@
 
     property bool vertical: (plasmoid.formFactor == PlasmaCore.Types.Vertical)
 
-    Layout.minimumWidth: Layout.maximumWidth
-    Layout.minimumHeight: Layout.maximumHeight
+    Layout.minimumWidth: units.iconSizes.small
+    Layout.minimumHeight: Layout.minimumWidth
 
     Plasmoid.preferredRepresentation: Plasmoid.fullRepresentation
     Plasmoid.onActivated: showdesktop.showDesktop();

